docs/introduction.html

```diff
@@ -18,7 +18,7 @@
 <script><!--#include virtual="js/templateData.js" --></script>
 
 <script id="introduction-template" type="text/x-handlebars-template">
-  <h3> Kafka is <i>a distributed streaming platform</i>. What exactly does that mean?</h3>
+  <h3> Kafka&trade; is <i>a distributed streaming platform</i>. What exactly does that mean?</h3>
```

Comment:
Checked with Vivian that we need to use the trademark on first reference of Kafka besides the logos, and it needs to be referenced as "Apache Kafka" for the first time. When it was not referring to the project itself, e.g. "Kafka Streams API" or "Kafka topics" then we do not need to do that.